releases/1.8/beta/kubeflow/bundle.yaml

```diff
@@ -145,84 +158,91 @@ applications:
     _github_repo_name: knative-operators
   kserve-controller:
     charm: kserve-controller
-    channel: latest/beta
+    channel: 1.8/beta
     scale: 1
     trust: true
     _github_repo_name: kserve-operators
   kubeflow-dashboard:
     charm: kubeflow-dashboard
-    channel: latest/beta
+    channel: 1.8/beta
     scale: 1
     trust: true
     _github_repo_name: kubeflow-dashboard-operator
   kubeflow-profiles:
     charm: kubeflow-profiles
-    channel: 1.7/edge
+    channel: 1.8/beta
     scale: 1
     trust: true
     _github_repo_name: kubeflow-profiles-operator
   kubeflow-roles:
     charm: kubeflow-roles
-    channel: latest/beta
+    channel: 1.8/beta
     scale: 1
     trust: true
     _github_repo_name: kubeflow-roles-operator
   kubeflow-volumes:
     charm: kubeflow-volumes
-    channel: latest/beta
+    channel: 1.8/beta
     scale: 1
     _github_repo_name: kubeflow-volumes-operator
   metacontroller-operator:
     charm: metacontroller-operator
-    channel: latest/beta
+    channel: 3.0/beta
     scale: 1
     trust: true
     _github_repo_name: metacontroller-operator
+  mlmd:
+    charm: mlmd
+    channel: 1.14/beta
+    scale: 1
+    _github_repo_name: mlmd-operator
   minio:
     charm: minio
-    channel: latest/beta
+    channel: ckf-1.8/beta
     scale: 1
     _github_repo_name: minio-operator
   oidc-gatekeeper:
     charm: oidc-gatekeeper
-    channel: latest/beta
+    channel: ckf-1.8/beta
     scale: 1
     trust: true
     _github_repo_name: oidc-gatekeeper-operator
   pvcviewer-operator:
     charm: pvcviewer-operator
-    channel: latest/beta
+    channel: 1.8/beta
     scale: 1
     trust: true
+    series: focal
```

Comment:
Is `series` required here?

Reply:
Resolving, as this is necessary as discussed in https://github.com/canonical/bundle-kubeflow/pull/733#discussion_r1400248494